helm/chaos-mesh/templates/post-install-mutatingwebhook-job.yaml

```diff
@@ -46,27 +47,11 @@ spec:
               apiVersion: admissionregistration.k8s.io/v1beta1
               kind: MutatingWebhookConfiguration
               metadata:
-                name: chaos-mesh-sidecar-injector
-                labels:
-                  app.kubernetes.io/name: {{ template "chaos-mesh.name" . }}
-                  app.kubernetes.io/managed-by: {{ .Release.Service }}
-                  app.kubernetes.io/instance: {{ .Release.Name }}
-                  app.kubernetes.io/component: admission-webhook
-                  helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+"  "_" }}
+                name: {{ template "chaos-mesh.mutation" . }}
               webhooks:
-                - name: admission-webhook.pingcap.com
+                - name: {{ template "chaos-mesh.webhook" . }}
                   clientConfig:
-                    service:
-                      name: chaos-mesh-controller-manager
-                      namespace: ${K8S_NAMESPACE}
-                      path: "/inject-v1-pod"
```

Comment:
Seems the `service` field is required here. I tried to deploy with `certManager.enabled = false` and I got the error below in `webhook-mw-job` pod,
```
# job failed
webhook-mw-job-4fpbb                        0/1     Error       0          69s
webhook-mw-job-89zxk                        0/1     Error       0          29s
webhook-mw-job-kv44l                        0/1     Error       0          99s
webhook-mw-job-lrb6z                        0/1     Error       0          89s
webhook-mw-job-qlfsj                        0/1     Error       0          101s


klo webhook-mw-job-kv44l    
The MutatingWebhookConfiguration "chaos-mesh-sidecar-injector" is invalid: webhooks[0].clientConfig: Required value: exactly one of url or service is required

```

Reply:
What is your k8s version? My version is v1.17.2 . System operating normally . I will fix it right now.

Reply:
I am using 1.17.0

Reply:
Can not recurrence this error,  if I use 1.17.0. But I have added the `service` field on the `MutatingWebhookConfiguration`. Please try again. Thank you.

Reply:
Thanks! With the updated PR, and then it works!